### DISCUSSION TITLE
New default explorer for Crab Network: https://crab-scan.darwinia.network/

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#152] [General]
https://crab-scan.darwinia.network/

https://crab.subscan.io/ will be deprecated to save cost.

**Comment by hackfisher** [comment]
Integration for chainlist 和 tally need to be updated.

**Reply by boundless-forest** [reply]
I don't know how to where to update the tally explorer link.  @hujw77 Please take a look.

**Comment by boundless-forest** [comment]
- https://github.com/darwinia-network/docs/pull/75
- https://github.com/ethereum-lists/chains/pull/5320
